crates/control/src/motion/motion_selector.rs

```diff
@@ -122,6 +124,7 @@ fn transition_motion(
         (_, _, MotionType::FallProtection, _) => MotionType::FallProtection,
         (_, _, MotionType::WideStance, _) => MotionType::WideStance,
         (_, _, MotionType::CenterJump, _) => MotionType::CenterJump,
+        (MotionType::ArmsUpStand, _, _, false) => MotionType::ArmsUpStand,
```

Comment:
Why is this required? Isnt this a default transition?

Reply:
If the robot is lifted up by the referee during arms up stand he goes back to stand which we do not want